src/dpp/message.cpp

```diff
@@ -882,6 +882,38 @@ bool attachment::is_remix() const {
 	return flags & a_is_remix;
 }
 
+time_t attachment::get_expire_time() const {
+	size_t attr_position = url.find('?');
+	/*If no attributes were sent in url, we do not need to parse more*/
+	if(url.npos == attr_position){
+		return 0;
+	}
+	std::string attributes = url.substr(attr_position+1);
+	std::vector<std::string> attr_list = utility::tokenize(attributes,"&");
```

Comment:
imo, you should put `url.substr(attr_position+1)` inside `utility::tokenize` as you don't use `attributes` anywhere else but this. However, I'd like other thoughts on this.

Reply:
please space stuff out, (attr_position + 1) not (attr_position+1)

Reply:
I generally like to add intermediate variables to make the code easier to follow, iirc compiler optimizes them away if able to